administrator-manual/en/packages.rst

```diff
@@ -5,72 +5,176 @@
+    development cycle, but all of them prefer software stability over bleeding
+    edge features.
+
+    Refer to the Community forum [#NSCOM]_ and :ref:`release-notes-section` for
+    more information about NethServer updates.
+
+    Updates released by the CentOS project are immediately available on
+    NethServer. More info about CentOS updates:
+
+    - https://wiki.centos.org/FAQ/General
+    - https://access.redhat.com/support/policy/updates/errata/
+    - https://access.redhat.com/security/updates/backporting
+    - https://access.redhat.com/security/
+
+    Updates released by EPEL are available only if the **unlocked** software
+    updates origin is selected, as explained below.
+
+    .. hint::
+
+        Even if the above projects strive for software stability, care is
+        necessary to check the updates fit well together. Every time the system
+        is going to be updated, **create a backup of the data and review the
+        updates changelog** to understand what is going to happen. If possible,
+        test the updates in a non-production system. For any doubt ask the
+        NethServer community forum! [#NSCOM]_
+
+    The first time the Software center page is opened, it is required to select
+    **the origin of software updates**. This decision can be changed at any
+    later time from :guilabel:`Software center > Configure`:
+
+    :dfn:`Unlocked`
+
+        The Software center considers updates from all available software
+        repositories. It ensures every installed package is at its latest
+        version.
+
+        When a new release of CentOS is available, a warning message is shown to
+        inform the user about the new upstream distribution release.
+
+    :dfn:`Locked`
+
+        The Software center prevents to install the updates coming from a CentOS
+        release different than the current system version. For example it
+        prevents to install updates from CentOS 7.6 if NethServer version is
+        7.5. Packages from other repositories which are generically compatible
+        with "7" (like EPEL) are considered only when new modules are installed;
+        once they have been installed they are never updated.
+
+        When the NethServer project releases a new version, the Software center
+        asks to start a :dfn:`system upgrade`.
+
+    .. hint::
+
+        Choosing the *right* updates origin depends on the system
+        administrator's needs. As a rule of thumb, the **unlocked** origin
+        receives more updates that correct bugs and security issues so it needs
+        more time to review them. On the other hand the **locked** origin
+        receives less updates so it is more predictable.
 
     .. warning::
 
-        Installed packages **cannot be removed from the Software Center!**
-        Please contact customer support if you need to remove an installed module.
+        Using ``yum update`` from the command line works like selecting the
```

Comment:
This isn't true, I'd say something like:

When yum is used from the command line, repositories which use only the major release (like EPEL), will be always enabled, regardless of selected policy.